core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2516,4 +2516,14 @@ package object config {
       .version("3.5.0")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val DECOMMISSION_MAX_RATIO =
+    ConfigBuilder("spark.decommission.maxRatio")
+      .doc("Max ratio of decommissioning executor of running executors. Decommission too many " +
+        s"executors at the same time with shuffle or rdd migration enabled " +
+        s"could hurt performance of shuffle fetch " +
+        s"as shuffle or rdd migration compete network and disk IO with shuffle fetch")
+      .version("3.5.0")
+      .doubleConf
```

Comment:
We should do value validation with `.check`.

Reply:
Added check.